static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -393,6 +398,37 @@ const PublishPreview = () => {
               </Button>
             )
           }
+
+          {
+            isRevision
+              ? (
+                <Button
+                  className="btn btn-link"
+                  type="button"
+                  variant="link"
+                  onClick={handleRevisions}
+                >
+                  Back To Revisions
```

Comment:
I do not see this Back to Revisions button in current MMT. I see that it is just routing back to revision table view. Was there a discussion about this button?

Reply:
There wasn't but I believe changing it to this solves two problems at once. Could I get this approved through Trevor?